.github/workflows/continuous-integration.yml

```diff
@@ -50,10 +50,10 @@ jobs:
       fail-fast: false
       matrix:
         swift:
-          - 5.5-amazonlinux2
-          - 5.6-focal
           - 5.7-amazonlinux2
-          - 5.8-jammy
+          - 5.7-focal
+          - 5.9-amazonlinux2
+          - 5.9-jammy
```

Comment:
Not entirely sure if this matters, but according to the ticket https://github.com/awslabs/aws-sdk-swift/issues/1133, we should have `5.7-jammy` as well. I'd assume `5.7-focal` here should be replaced with `5.7-jammy`. @jbelkins could you confirm?